.vscode/launch.json

```diff
@@ -64,87 +61,87 @@
             "type": "python",
             "request": "launch",
             "program": "pbsync/pbsync.py",
-            "console": "integratedTerminal",
+            "console": "integratedTerminal"
         },
         {
             "name": "Sync All",
```

Comment:
While running sync all, after engine extraction, I'm getting that error:

```
Exception has occurred: FileNotFoundError
[WinError 2] The system cannot find the file specified
  File "C:\Users\dyanikoglu\Desktop\PBSync\pbpy\pbtools.py", line 25, in run_with_combined_output
    return subprocess.run(*cmd, text=True, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
  File "C:\Users\dyanikoglu\Desktop\PBSync\pbpy\pbtools.py", line 29, in get_combined_output
    return run_with_combined_output(*cmd).stdout
  File "C:\Users\dyanikoglu\Desktop\PBSync\pbpy\pbunreal.py", line 197, in check_ue4_file_association
    file_assoc_result = pbtools.get_combined_output(["assoc", uproject_ext])
  File "C:\Users\dyanikoglu\Desktop\PBSync\pbsync\pbsync.py", line 152, in sync_handler
    if pbunreal.check_ue4_file_association():
  File "C:\Users\dyanikoglu\Desktop\PBSync\pbsync\pbsync.py", line 336, in main
    sync_handler(args.sync, args.repository, args.bundle)
  File "C:\Users\dyanikoglu\Desktop\PBSync\pbsync\pbsync.py", line 355, in <module>
    main()
```

Log:
```
...
2020/05/10 19:09:18 Extracting editor (165441/165464)...
2020/05/10 19:09:19 Extracting editor (165459/165464)...
2020/05/10 19:09:21 Extracting editor (165460/165464)...
2020/05/10 19:09:25 Extracting editor (165463/165464)...
2020/05/10 19:09:25 Extracting editor (165464/165464)...
2020/05/10 19:09:25 Registering ue4v:4.24-PB-20200412
10-May-20 19:09:26 [INFO ]  Engine build editor-4.24-PB-20200412 successfully registered
10-May-20 19:09:26 [INFO ]  ------------------
```

Reply:
Should be fixed now